### DISCUSSION TITLE
征集新功能

### DISCUSSION THREAD

**Discussion by rafa0128** [smart-test-ti/SoloX#56] [Ideas]
SoloX诚挚征集新功能，如果你有好的想法，请在这里留言！

**Comment by rafa0128** [comment]
🥳🥳🥳🥳🥳

**Comment by cooltaby** [comment]
Wireless connection support for iOS like Android.
Login, account system to access the tool over the internet from any location.

running Screenshots on dashboard for each frame like graph of fps

**Comment by zhangbb1991** [comment]
收集的性能数据支持自定义存储位置，并可以使用命令行生成html格式的报告。参考allure。
